DER/SecurityBasedDerivatives/EquityForwards.rdf

```diff
@@ -88,34 +83,16 @@
 		<rdfs:subClassOf rdf:resource="&fibo-der-sbd-sbd;EquityDerivative"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/DerivativesBasics/hasUnderlier"/>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
 				<owl:someValuesFrom rdf:resource="&fibo-der-sbd-sbd;EquityObservable"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasBuyer"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-sbd-eqf;EquityForwardBuyer"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-fwd;hasSeller"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-sbd-eqf;EquityForwardSeller"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-der-sbd-eqf;dividendAdjustment"/>
 				<owl:someValuesFrom rdf:resource="&fibo-der-sbd-eqf;DividendAdjustmentPeriod"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-sbd-eqf;hasConditions"/>
-				<owl:someValuesFrom rdf:resource="&fibo-der-sbd-eqs;DividendConditions"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
```

Comment:
Better to have no skos:definition which will be flagged. Having "Definition needed" is not acceptable since it subverts the hygiene test and the need to supply one can easily be overlooked.

Reply:
I agree - we haven't gone through this in its entirety, just fixed it partially, but it's far better than it was before